packages/slate-types/src/nodes/ImageNode.ts

```diff
@@ -11,16 +12,17 @@ export enum ImageLayout {
     FULL_WIDTH = 'full-width',
 }
 
-export interface ImageNode extends ElementNode {
+export interface ImageNode extends ElementNode, Alignable {
     type: typeof IMAGE_NODE_TYPE;
     /** caption */
     file: UploadcareImageStoragePayload;
     /** empty string if no URL */
     href: string;
     layout: ImageLayout;
+    align: Alignment;
     new_tab: boolean;
-    /** matches this regexp: /^\d+(\.\d+)?%$/ */
-    width: string;
+    /** matches this regexp: /^\d+(\.\d+)?(%|px)$/ */
+    width: `${number}%` | `${number}px`;
```

Comment:
why not as `SizeString`?

Reply:
You are right that this is the same type, but I didn't want to couple `Size` library module (`slate-editor` package) and this generic `ImageNode` interface (`slate-types` package).

I've deduplicated the type definition by exporting the `ImageWidth` type from the `@prezly/slate-types` package.

Reply:
oh, I haven't seen this is in two different packages. My bad :)